docs/gen-1/red-blue/main-glitchless/beginner-route/README.md

```diff
@@ -3,9 +3,7 @@
 * [Lass Version](lass/)
 * [No Lass Version](nolass/)
 
-- Credit to Squarerootofsheep for this adaptation from the main route.
-
 ## What are the differences?
 - These differences revolve around fighting an extra Lass on route 3 for a much safer Moon Rocket fight and a lot of improved ranges.
 - The Moon Rocket fight can be complicated to play "safely", so the Lass route makes the fight much safer and simpler.
-- The Lass route has a strict requirement of using [Entr's Moon Manip](https://pastebin.com/jnj9j47S), which is slightly worse than [Route 3 Moon Manip](https://pastebin.com/tggXpQRC). However, the differences are rather minimal and both are viable, even at the top level.
```

Comment:
I might say "starting out it may be easier to learn the shorter moon manip while using the Lass Route"

Reply:
The goal of this part was to say that new runners shouldn't focus on learning any of those 2 manips (as we often see people getting stuck trying to learn a full moon manip before they even start to play the game), I'm not sure what is the best way to phrase it